inst/odin/carehomes.R

```diff
@@ -42,7 +42,9 @@ n_I_P_vaccinated[, ] <-
   sum(n_II_P_next_vacc_class[i, , , j])
 dim(n_I_P_vaccinated) <- c(n_groups, n_vacc_classes)
 n_R_vaccinated[, ] <-
-  sum(n_R_next_vacc_class[i, , j]) + sum(n_RS_next_vacc_class[i, , j])
+  sum(n_RR_next_vacc_class[i, , j]) +
+  sum(n_RS_next_vacc_class[i, , j]) +
+  sum(n_RE_next_vacc_class[i, , j])
```

Comment:
not sure how you have defined those but looks like the n_RE should come from another strain so should perhaps be indexed differently? Please check